bindings/python/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+find_package(PythonLibs REQUIRED)
+include_directories(${PYTHON_INCLUDE_PATH})
+
+# Build python bindings
+set(CMAKE_SWIG_FLAGS "")
+set_property(SOURCE ../uast.i PROPERTY CPLUSPLUS ON)
```

Comment:
This relative references gives me the impression that local compilation should work, yet it fails because `libuast/node.h` is not found.